Example/Tests/SHA256Tests.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  SHA256Tests.swift
+//  Krypt_Tests
+//
+//  Created by Simon Feistel on 21.01.20.
+//  Copyright © 2020 CocoaPods. All rights reserved.
+//
+
+import Krypt
+import XCTest
+
+class SHA256Tests: XCTestCase {
+  // Computed using `openssl dgst -sha256 -hex [file path]
+  private let expectedHash = "4b9ecf8a918594442fa56d14f2f8e975b258d134bf71c3a05c913b952d40af46"
+
+  func testSAH256Calculation_nonBuffered__shouldProduceCorrectHash() {
```

Comment:
Would appreciate following our naming convention: 
```
// ------------------------------------------------------------------------------------------------
//
// Test functions
//
// ------------------------------------------------------------------------------------------------
// Format:
func testWhatIsBeingTested_conditions_additionalCondition__expectedResult() {}

// Example:
func testSectionJournal_multipleSnapshots_noTodaySnapshot__shouldTakeLatestAsTodaysEntry() {}
```
```suggestion
  func testDigestData__shouldProduceExpectedHash() {
```